src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx

```diff
@@ -13,50 +13,57 @@ redirects:
   - /docs/what-entity-new-relic
 ---
 
-New Relic monitoring is built around the concept of the **entity**. An entity is anything that reports data to New Relic. This document explains:
+New Relic monitoring is built around the concept of **entities**, which are basically the hosts and services and other systems that New Relic monitors. In this doc, you'll learn how we define entities, what you can do with them, and how you can create your own entities or groupings of entities. 
```

Comment:
I prefer the original phrasing. We like to emphasize the fact that it's literally *anything* that reports data, we don't want to start the doc by narrowing it down to existing components. Just my opinion, obviously.

Reply:
it's a little more vague than even that, though, which is part of the problem I was having. For example: an entity can be a dashboard or workload (and soon some other things, like a github repo), which aren't technically things that report data to New Relic. So I was trying to find a way to say 'it's basically that' even though it's not. But I"ll work on it some more.

Reply:
You're right. How about "An entity is anything that generates data and New Relic can monitor".